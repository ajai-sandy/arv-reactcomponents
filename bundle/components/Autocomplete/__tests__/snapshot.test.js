import React from "react";
import { shallow, mount } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import TestComponent from "../test.component";

describe("Root Component Tests", () => {
  it("Matches Shallow SnapShot", () => {
    expect(shallowToJson(shallow(<TestComponent />))).toMatchSnapshot();
  });

  it("Matches Mounted SnapShot", () => {
    expect(mount(<TestComponent />)).toMatchSnapshot();
  });
});
