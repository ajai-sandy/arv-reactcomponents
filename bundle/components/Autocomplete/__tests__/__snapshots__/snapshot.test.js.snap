// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Component Tests Matches Mounted SnapShot 1`] = `
<AutocompleteTest>
  <Autocomplete
    className=""
    getSelection={[Function]}
    inpList={
      Array [
        "Arrow",
        "Levis",
        "Hidden",
        "Gap",
      ]
    }
    minTextLength={0}
    renderInput={[Function]}
    renderList={[Function]}
    selectedIndex={-1}
  >
    <div
      className="nwc-autocomplete "
    >
      <Input
        checked={null}
        className=""
        disabled={false}
        id="autocomplete-input"
        inputRef={[Function]}
        maxLength={null}
        name={null}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Enter text"
        type="text"
        validateWithPattern={null}
        value=""
      >
        <input
          checked={null}
          className="nwc-inp  "
          disabled={false}
          id="autocomplete-input"
          maxLength={null}
          name={null}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          placeholder="Enter text"
          tabIndex="0"
          type="text"
          value=""
        />
      </Input>
    </div>
  </Autocomplete>
</AutocompleteTest>
`;

exports[`Root Component Tests Matches Shallow SnapShot 1`] = `
<Autocomplete
  className=""
  getSelection={[Function]}
  inpList={
    Array [
      "Arrow",
      "Levis",
      "Hidden",
      "Gap",
    ]
  }
  minTextLength={0}
  renderInput={[Function]}
  renderList={[Function]}
  selectedIndex={-1}
/>
`;
